src/App.tsx

```diff
@@ -1,203 +1,108 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import { TodoItem } from './components/TodoItem';
+import { TodoList } from './components/TodoList';
+import { FilterStatus } from './types/FilterStatus';
+import { TodoCreatingForm } from './components/TodoCreatingForm';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState<FilterStatus>(
+    FilterStatus.All,
+  );
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    getTodos()
+      .then(data => {
+        setTodos(data);
+      })
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [errorMessage, setErrorMessage] = useState('');

  useEffect(() => {
    setIsLoading(false);
    
    getTodos()
      .then(setTodos)
```
You should set loader when you starting to loading something